code-studio/src/js/code-studio.js

```diff
@@ -2,7 +2,56 @@
  * @file Replacement for application.js.erb in dashboard as much as possible.
  *       Code added here will be loaded for (almost) every page in dashboard,
  *       so use sparingly!
+ *
+ * Note: This is included _after_ application.js.erb during our transition period,
+ * so when moving things preserve the include order as much as possible.
  */
 'use strict';
+/* global Sounds */
 
 require('./videos');
+window.React = require('react');
+require('./components/*.jsx', {mode: 'expand'});
```

Comment:
That makes sense.  I'll go ahead and expand this glob now.  That way we remove the `require-globify` dependency/transform and we're set up to de-global these components one at a time.